### PR TITLE
Add build script

### PR DESCRIPTION
**Pull request by ImMaax**

### CODE REVIEW

**Comment by Chirag-Khandelwal** [comment]
Just a couple (possibly final) things.
1. Could you rename `Automated building` to `Automated Build` and `Manual building` to `Manual Build` in README?
2. We also need to check (like `git`) if `cmake` and `sudo` exist.
Otherwise, looks pretty great!

**Comment by ImMaax** [comment]
Added checks for CMake and sudo and updated the README!

**Comment by Chirag-Khandelwal** [comment]
Awesome! Thanks!! ❤️❤️
Merging this, and closing #10

File: build.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env sh
+echo '[INFO] Feral Installer - (c) Copyright 2020 Maximilian Götz (https://www.maxbits.net/)'
+echo '[INFO] Please make sure that Git, CMake, LibGMP and LibMPFR are installed!'
+echo '[INFO] Installation started. Checking for Git, sudo and CMake...'
+
+# Check for Git
+if hash git 2>/dev/null; then
+    echo '[INFO] Git found.'
+else
+    echo '[FATAL] Git not available! Install it to continue! Aborting...'
```
Comment (vitkarpov) [inline build.sh:10]: I'm not an expert but shouldn't it be **is** here and down below?

Comment (ImMaax) [reply build.sh:10]: You're right, but I left them out intentionally to make the messages a little shorter. If you think that the *is* is important, feel free to add it! :)

**Comment by Chirag-Khandelwal** [comment]
@ImMaax I did some modifications and additions - mainly regarding number of cores used as well as when sudo will be used.
Updated it in the repo. Do check it out and lemme know what u think 😁

**Comment by ImMaax** [comment]
@Electrux Wow, really some great improvements!
One thing I'd probably change is how functions are defined:
One is defined like this:
```sh
function fnname {}
```
And another one is defined like this:
```sh
fnname {}
```
In my opinion, it doesn't matter which style is being used, but we should agree on one of the two. Otherwise, it'll most likely get inconsistent pretty quickly.
But, again, great work! :)

**Comment by Chirag-Khandelwal** [comment]
I'd go with the first style actually, just because it's more explicit and easier to see in long(er) scripts.
Especially helpful when combined with syntax highlighting 🤔
What say?

**Comment by ImMaax** [comment]
Yep, let's do it this way. Makes code easier to read, probably

**Comment by Chirag-Khandelwal** [comment]
Okie, I'll update the script accordingly then.

**Comment by Chirag-Khandelwal** [comment]
btw, apparently, `function` before function name is actually just a bashism, not posix compliant 😂.
Back to
```bash
function_name() {}
```
style it is.

**Comment by ImMaax** [comment]
Lol, that's interesting! Didn't know about this

**Comment by Chirag-Khandelwal** [comment]
Yea, me neither! 😂

File: build.sh (hunk 1 above)
Comment (vitkarpov) [reply build.sh:10]: Nah, I don't. Just noticed, that's cool! 👍

**Pull request merged**